packages/core/test/feedback/FeedbackFormManager.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { logger } from '@sentry/core';
+import { render } from '@testing-library/react-native';
+import * as React from 'react';
+import { Text } from 'react-native';
+
+import { FeedbackFormProvider, showFeedbackForm } from '../../src/js/feedback/FeedbackFormManager';
+
+jest.mock('../../src/js/feedback/utils', () => ({
+  isModalSupported: jest.fn(),
+}));
+
+beforeEach(() => {
+  logger.error = jest.fn();
+});
+
+describe('FeedbackFormManager', () => {
+  it('showFeedbackForm displays the form when FeedbackFormProvider is used', () => {
+    require('../../src/js/feedback/utils').isModalSupported.mockReturnValue(true);
```

Comment:
l: `import { isModalSupported } from ''../../src/js/feedback/utils'';` and then `(isModalSupported as jest.MockedFunction<typeof isModalSupported>)` should also work and it will be typesafe.

Reply:
Good idea Krystof 👍 
Updated with d8ab9146f4fc130dd1a5b5c1c46af6fc8b98a831